src/site/static/benefits-eligibility/eligibility.liquid

```diff
@@ -281,6 +281,9 @@ pageClass: "page-benefits-eligibility"
           {% option "shelter" %}
             I live in a shelter, interim housing, or emergency housing
           {% endoption %}
+          {% option "unhoused-risk" %}
```

Comment:
Much like how it [used to be set up](https://github.com/theunitedeffort/theunitedeffort.org/pull/487/files#diff-57c51b14537c1c639a55767fed4cefb7dd53846895b692c98f19e4188c0dee70L289-L302) before it was removed, this question should go in the `household-housed` page so that it only gets asked if the person notes that they are currently housed.

Reply:
I'm not sure how the `household-housed` HTML tags are referred to in eligibility.js. Previously when it was under the `household-situation` page the HTML tag for `unhoused-risk` could just be referred to in eligibility.js. Similar to the following:
```
  const isUnhoused = isOneOf(input.housingSituation, [
    'vehicle',
    'transitional',
    'hotel',
    'shelter',
    'no-stable-place']);
```
That doesn't seem to be the case with `household-situation`.

I added `    unhousedRisk: getValueOrNull('unhoused-risk'),` to `const inputData` and then used 
```
  program.addCondition(new EligCondition(
    'Be at risk of losing your housing',
    input.unhousedRisk));
```

I don't know if this is the right way to do it.

Reply:
I'm not fully sure I understand your question, but I'll give it a go.

The form question you are adding will be a yes/no question so you can use the custom Liquid shortcode `yesNo` as you can see in this [example](https://github.com/theunitedeffort/theunitedeffort.org/blob/edaca0e2879c7f4690dd39f7bcfd61f3608c421f/src/site/static/benefits-eligibility/eligibility.liquid#L297) (and also the old change linked above when this question was originally removed). The parameter you pass into `yesNo` is the `id` of the generated HTML element.  So whatever string you put there is what you will query when building `inputData`.  You can see an example [here](https://github.com/theunitedeffort/theunitedeffort.org/blob/edaca0e2879c7f4690dd39f7bcfd61f3608c421f/src/site/_includes/js/eligibility.js#L2753)

What you said you did regarding `unhousedRisk` and how to use it looks ok to me.  It's similar to this [example](https://github.com/theunitedeffort/theunitedeffort.org/blob/edaca0e2879c7f4690dd39f7bcfd61f3608c421f/src/site/_includes/js/eligibility.js#L1972). Is it not working?